src/components/common/Footer.tsx

```diff
@@ -0,0 +1,95 @@
+import { FC } from 'react';
+import Link from 'next/link';
+
+import { Logo } from './Logo';
+import { Icon } from './Icon';
+import { isExternalUrl } from '../../utils/helpers';
+import config from '../../config';
+import * as routes from '../../routes';
+
+const content = {
+  note: (
+    <p className="mt-2">
+      Made with 💜 by{' '}
+      <Link href={config.raydiantUrl}>
+        <a
+          className="hover:text-slate-700 dark:hover:text-slate-300"
+          target="_blank"
+          rel="noreferrer"
+        >
+          Raydiant
+        </a>
+      </Link>
+    </p>
+  ),
+  menus: [
+    {
```

Comment:
Card to track this work here: https://trello.com/c/6wiUCVvl/4345-developer-docs-site-populate-footer-with-content